tark-refseq-loader/conf/refseq_source.ini

```diff
@@ -1,6 +1,7 @@
 [DEFAULT]
 source=2
 shortname=109_20190905
+release_date=xxxx-xx-xx
```

Comment:
This release date will change based on `shortname` right?

Reply:
@bilalebi Yes. All the values in the config file change based on the release. The values you see there at the moment are just placeholders. I have now added a comment now to make it clear.